src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -400,13 +400,21 @@ def test_fir_filter_coefficients():
 '''
 
 
-'''
-def test_send_software_edge_trigger(session):
+def test_send_software_edge_trigger_start(session):
```

Comment:
The test doesn't do much. You could just `session.send_software_edge_trigger()` and know nothing more.
If you want to actually see if the trigger is received then you need to configure the device for software trigger, initate, send the trigger, then wait until done.

Reply:
We're not testing the driver. I just want to make sure the function call doesn't error.

Reply:
It won't error, but are we calling it correctly? Maybe we are sending the wrong trigger...